packages/web-console/src/scenes/Schema/Table/index.tsx

```diff
@@ -221,9 +239,40 @@ const Table = ({
               />
             )
           },
-        },
-      ],
+        }
+        const children: TreeNode[] = []
+
+        if (matView && matViewData) {
+          children.push({
+            name: "Base Table",
+            render: () => (
+              <Row
+                table_id={id}
+                kind="info"
+                name="Base Table"
+                value={matViewData?.base_table_name}
+              />
+            )
+          })
 
+          const response = await showMatViewDDL(table_name)
+          if (response && response.data && response.data.length > 0) {
+            children.push({
+              name: "DDL",
+              render: () => (
+                <Row
+                  table_id={id}
+                  kind="info"
+                  name="DDL"
```

Comment:
```suggestion
                  name="Query"
```

Perhaps more simple language for less technical personas?

DDL is correct and also jargon.

Up to you - we can use DDL in code, but maybe soften for viewer exposure.

Reply:
Changed it to "Query" 👍